wasmbinding/bindings/msg.go

```diff
@@ -100,6 +100,11 @@ type SetMarketEnabled struct {
 	Enabled bool   `json:"enabled"`
 }
 
+type UpdateCollateral struct {
+	Denom           string `json:"denom"`
+	ContractAddress string `json:"contract_address"`
+}
+
```

Comment:
Instead of using the `wasmbinding` folder, it simply add the keeper call registered to the message under the `Admin` field, gated by the `SudoKeeper`

Reply:
Addressed by https://github.com/NibiruChain/nibiru/pull/1656/commits/2dd5807e12f4c1b99a5c9f62226ff8c5a984140d